packages/table/src/V11Adapter/V11Adapter.tsx

```diff
@@ -142,9 +169,15 @@ const V11Adapter = <T extends LGRowData>({
                       // index by row.index (not the index of the loop) to get the sorted order
                       // @ts-expect-error `processedData` is structured to be indexable by `row.index`
                       processedData[row.index]?.[cell.column.id]?.();
+                    const {
+                      children,
+                      isHeader,
```

Comment:
Are `isHeader` and `isDisabled` the only props that need filtering out?

Reply:
<img width="715" alt="image" src="https://github.com/mongodb/leafygreen-ui/assets/17347644/4bdfb47b-8be4-4916-b70d-443be7e6ce47">